client/lib/products-values/index.js

```diff
@@ -203,6 +203,8 @@ export function isJetpackMonthlyPlan( product ) {
 }
 
 export function isJetpackProduct( product ) {
+	// When we update the jetpack products list.
```

Comment:
This comment is not in the right place. It should be in `isJetpackBackup`, because that's the function we want to update when there are other products.

Reply:
True, but the problem is that when we update `isJetpackProduct` to add or remove new products it will break `isJetpackBackup`. Which is what I try to avoid. 
Someone that is updating `isJetpackProduct` will not know to update `isJetpackBackup` to make sure that it doesn't break things. I am hoping a comment here will prevent this from happening.

Reply:
That's fair. Let's fix the typos then 😛